src/one_dragon/base/controller/pc_screenshot/bitblt_screencapper.py

```
@@ -0,0 +1,217 @@
+        with self._lock:
+            if self.hwndDC is None or self.mfcDC is None:
+                if not self.init():
+                    return None
+
+            # 获取屏幕尺寸
+            screen_width = ctypes.windll.user32.GetSystemMetrics(0)
+            screen_height = ctypes.windll.user32.GetSystemMetrics(1)
+
+            screenshot = self._capture_with_retry(0, screen_width, screen_height)
+            if screenshot is None:
+                # 如果第一次失败，尝试重新初始化并重试一次
```

Ruff (0.14.3)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)